### PR TITLE
Update Docs: Enable-billing-3.1.1

### PR DESCRIPTION
**Pull request by serenashe**
Signed-off-by: adakisme <serenashe@yunify.com>
/assign @Patrick-LuoYu

### CODE REVIEW

**Comment by serenashe** [comment]
Could you please help review this PR? Many thanks. @Felixnoo

File: content/en/docs/toolbox/metering-and-billing/enable-billing.md [hunk 1]
```diff
@@ -52,12 +40,47 @@ Perform the steps below to enable KubeSphere Billing.
...
+         <th>Parameter</th>
+         <th>Description</th>
+       </tr>
+       <tr>
+         <td><code>retentionDay</code></td>
+         <td><code>retentionDay</code> determines the date range displayed on the <b>Metering and Billing</b> page for users. The value of this parameter must be the same as the value of <code>retention</code> in <a href='../../../faq/observability/monitoring/#how-to-change-the-monitoring-data-retention-period'>Prometheus</a>.</td>
+       </tr>
+       <tr>
+         <td><code>currencyUnit</code></td>
+         <td>The currency that displays on the <b>Metering and Billing</b> page. Currently allowed values are <code>CNY</code> (Renminbi) and <code>USD</code> (US dollars). If you specify other currencies, the console will display cost in USD by default.</td>
```
Comment (Felixnoo) [inline content/en/docs/toolbox/metering-and-billing/enable-billing.md:57]: `that displays` -> `that is displayed`

Comment (serenashe) [reply content/en/docs/toolbox/metering-and-billing/enable-billing.md]: Changed.

**Pull request merged**